### ISSUE TITLE
Electrode/channel locations reporting improvement

### ISSUE THREAD

**Issue by asoskic** [INCF/artem-is#95]
At the moment, people describe their electrode locations by linking to an external document where this information is available (e.g., preprint, preregistration, etc), they describe referencing and re-referencing, and taken together, these three can be used to understand which channels there are in a dataset. In the section on data analysis, they list channels that were used and describe verbally if and how they were aggregated into ROI-s.

In the future, it would be nice to have an option to select a cap, and then select/deselect electrodes to show the exact recording montage, and also to describe data analysis by selecting/deselecting channels. This is too ambitious for Version 1, though, so we are leaving this as an idea for the future.

**Comment by sanchalitorpe-source** [comment]
This issue is addressed by PR #152, which introduces a multi-select electrode list to improve electrode/channel reporting.

More advanced cap-based selection is intentionally left out of scope for Version 1. Closing this issue once the PR is merged.
